tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -38,7 +38,7 @@ export class Ide {
 
     constructor(
         @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper,
-        @inject(CLASSES.NotificationCenter) private readonly notificationCenter: NotificationCenter
+        @inject(CLASSES.NotificationCenter) private readonly notificationCenter: NotificationCenter,
```

Comment:
@makambalaji this doesn't look right. There are no more variables after this one, the `,` shouldn't be there.